test/expected/pg_dump.out

```diff
@@ -41,7 +40,7 @@ SELECT count(*)
      AND refobjid = (SELECT oid FROM pg_extension WHERE extname = 'timescaledb');
  count 
 -------
-   206
+   100
```

Comment:
Means there are less objects now. This is just to make sure same as above. expected.